extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -256,21 +187,33 @@ define('ext.wikia.adEngine.template.porvata', [
 				});
 			}
 
+			if (typeof params.onReady === 'function') {
+				// TODO remove this when Veles 1.0 is killed
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)